src/test-helpers/e2e-helpers/initPage.ts

```diff
@@ -0,0 +1,32 @@
+import skipTutorialScreen from './skipTutorialScreen'
+import { Browser, Device, Page } from 'puppeteer'
+
+export interface InitPageOptions {
+  puppeteerBrowser?: Browser,
+  url?: string,
+  skipTutorial?: boolean,
+  emulatedDevice?: Device,
+}
+
+/**
+ * Create a new incognito context and page.
+ */
+const initPage = async ({ puppeteerBrowser = browser, url = 'http://localhost:3000', emulatedDevice, skipTutorial = true }: InitPageOptions) => {
```

Comment:
Options argument should also be optional so that we can call `initPage()` instead of `initPage({})`.